### ISSUE TITLE
WARNING: API 'variant.getMergeAssets()' is obsolete and has been replaced with 'variant.getMergeAssetsProvider()'.

### ISSUE THREAD

**Issue by TheZoker** [chaquo/chaquopy#89]
Hi there,

As soon as I add the `apply plugin: 'com.chaquo.python'` line, I get this error on gradle built:
```
WARNING: API 'variant.getMergeAssets()' is obsolete and has been replaced with 'variant.getMergeAssetsProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getMergeAssets(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.
Affected Modules: app
```

**Comment by mhsmith** [comment]
In your top-level `build.gradle` file, what is the version of `com.android.tools.build:gradle`?

**Comment by TheZoker** [comment]
The version is:
`classpath 'com.android.tools.build:gradle:3.3.1'` (Android Studio 3.3)

I also tried
`classpath 'com.android.tools.build:gradle:3.4.0-beta03'` (Android Studio 3.4-beta)

Both did not work

**Comment by mhsmith** [comment]
OK, we've only tested Chaquopy up to Android Studio 3.2 so far, but we'll extend this to version 3.3 in the next release.

When you say "did not work", did the build actually fail? Or did it just display the warning?

**Comment by TheZoker** [comment]
No true, it's actually a warning.  So should be fine, right?

**Comment by mhsmith** [comment]
Hopefully. If you find any other problems related to Android Studio 3.3, please add them to this issue.

**Comment by mhsmith** [comment]
Warning fixed in Chaquopy 6.0.0.
